surveys/2021-annual-survey/questions.md

```diff
@@ -398,10 +398,10 @@ Activities:
 - Comment on, contribute to discussion of, or provide edits to an open RFC
 - Create a new thread or comment on internals.rust-lang.org
 - Discuss the Rust project in an official chat (either Zulip or Discord)
-- Open an issue on any repo in the rust-lang GitHub org
+- Open an issue on any repo in the rust-lang GitHub organisation
```

Comment:
nit: we've been using American English (probably due to me being the main author). I think this should be spelled "organization" then.